oauthdoctor/diag/config.go

```diff
@@ -446,17 +543,17 @@ func GetConfigFile(lang, overridePath string) (ConfigFile, error) {
 	return ConfigFile{
 		Filepath: filepath.Dir(overridePath),
 		Filename: filepath.Base(overridePath),
-		Lang:     lang}, nil
+		Lang:     lang}
 }
 
 // GetDefaultConfigFile returns the default config path of Google Ads API client
 // library.
-func GetDefaultConfigFile(lang string) (ConfigFile, error) {
+func GetDefaultConfigFile(lang string) ConfigFile {
 	var cfg ConfigFile
 
 	usr, err := user.Current()
 	if err != nil {
-		return cfg, err
+		log.Fatalf("Cannot get default config file: %s", err)
```

Comment:
I'm not sure what get means here. Consider a different prefix to the error statement.